apps/deploy-web/tests/ui/fixture/Leap/async/3672.js

```
@@ -862,44 +1451,166 @@ _global.SENTRY_RELEASE = { id: "0.22.9" };
+        ]
+      ]);
+      var l = Object.defineProperty,
+        o = Object.defineProperties,
+        r = Object.getOwnPropertyDescriptors,
+        d = Object.getOwnPropertySymbols,
+        c = Object.prototype.hasOwnProperty,
+        u = Object.prototype.propertyIsEnumerable,
+        m = (e, n, t) => (n in e ? l(e, n, { enumerable: !0, configurable: !0, writable: !0, value: t }) : (e[n] = t)),
+        v = (e, n) => {
+          for (var t in n || (n = {})) c.call(n, t) && m(e, t, n[t]);
+          if (d) for (var t of d(n)) u.call(n, t) && m(e, t, n[t]);
```

Biome (2.1.2)

[error] Shouldn't redeclare 't'. Consider to delete it or rename it.

't' is defined here:


(lint/suspicious/noRedeclare)